app/controllers/authentications_controller.rb

```diff
@@ -35,7 +35,12 @@ def destroy
         # Bryan: TESTED
         flash[:alert] = 'Bad idea!'
       elsif @authentication.destroy
-        flash[:notice] = 'Successfully removed profile.'
+	user = User.find(current_user.id)
+	if user.update_attributes(github_profile_url: nil)
+          flash[:notice] = 'Successfully removed profile.'
+	else
+          flash[:notice] = 'Github profile url could not be removed.'
```

Comment:
@tansaku 
line 45: wasn't reached. It covers the case when update_attributes fails to set github_profile_url as nil. This scenario seems rather not quite predictable in acceptance tests

line 48: wasn't reached. It covers sad case when github_profile_url is present, but no github authentication. I'm not sure how we can have the scenario can happen. 

Being unsure on both of these, i thought removing them might be a risk.